packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -47,23 +39,30 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
     this._updateEventHandlers(nextProps);
   }
 
-  public componentWillUnmount(): void {
-    const { ignoreExternalFocusing } = this.props;
+  public componentDidUpdate(prevProps: IFocusTrapZoneProps) {
+    const prevForceFocusInsideTrap = prevProps.forceFocusInsideTrap !== undefined ? prevProps.forceFocusInsideTrap : true;
+    const newForceFocusInsideTrap = this.props.forceFocusInsideTrap !== undefined ? this.props.forceFocusInsideTrap : true;
+
+    if (!prevForceFocusInsideTrap && newForceFocusInsideTrap) {
+      // Transition from forceFocusInsideTrap disabled to enabled. Emulate what happens when a FocusTrapZone gets mounted
+      FocusTrapZone._focusStack.push(this);
+      this._bringFocusIntoZone();
+    } else if (prevForceFocusInsideTrap && !newForceFocusInsideTrap) {
+      // Transition from forceFocusInsideTrap enabled to disabled. Emulate what happens when a FocusTrapZone gets unmounted
+      FocusTrapZone._focusStack = FocusTrapZone._focusStack.filter((value: FocusTrapZone) => {
+        return this !== value;
+      });
+      this._returnFocusToInitiator();
+    }
```

Comment:
This block of code is exactly the same as what's in componentWillUnmount. This should be pulled out into a helper function that both places call